src/docs/sdk/sessions.mdx

```diff
@@ -245,7 +247,7 @@ These sessions roughly correspond to HTTP requests or RPC calls in a
 server setting.
 
 - high in volume, typically one session for each request
-- the number of sessions is usually higher than the number of events
+- the number of sessions is usually higher than the number of captured events
```

Comment:
Sorry - not trying to beat a dead horse here, but captured _session_ events or captured sentry events in general? 

I guess I'm still not clear why this information is pertinent. Is what we're trying to say essentially, "you probably won't capture every request as a session and will probably only send one session event per session" and below, "you'll probably capture every session and each session will likely have multiple session events sent?"

Reply:
with `events`, I always mean sentry events, otherwise I would call them "session updates".

This is basically to highlight that you want to create a session for every request (maybe even 10k req/s), but not every request will raise an error, and thus a `capture_event`.